betaalpagina.php

```diff
@@ -38,7 +38,9 @@
     $id = GetCustomerID($_SESSION["email"]);
     if (Order($_SESSION["credentials"], $_SESSION['cart'],$id)) {
         header("location: transactie.php");
-    } else {
```

Comment:
hier lijk je twee keer de zelfde if uit te voeren. Alleen de tweede keer in een else.